libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -372,10 +371,24 @@ void CompilerUtils::convertType(Type const& _typeOnStack, Type const& _targetTyp
 			);
 			//shift all integer bits onto the left side of the fixed type
 			FixedPointType const& targetFixedPointType = dynamic_cast<FixedPointType const&>(_targetType);
-			if (auto typeOnStack = dynamic_cast<IntegerType const*>(&_typeOnStack))
-				if (targetFixedPointType.integerBits() > typeOnStack->numBits())
-					cleanHigherOrderBits(*typeOnStack);
-			solAssert(false, "Not yet implemented - FixedPointType.");
+			if (auto intType = dynamic_cast<IntegerType const*>(&_typeOnStack))
+			{
+				if (targetFixedPointType.integerBits() > intType->numBits())
+					cleanHigherOrderBits(*intType);
+				u256 shiftFactor = u256(256) << (targetFixedPointType.fractionalBits() - 8);
```

Comment:
I don't remember why I put that in there, but I remember the numbers didn't line up properly unless I did it this way...but let me go and recheck.